### PR TITLE
Made QuerySet.bulk_update() raise an error when batch_size is zero.

### PR DESCRIPTION
**Pull request by ebram96**

### CODE REVIEW

File: tests/queries/test_bulk_update.py [hunk 1]
```diff
@@ -125,6 +125,7 @@ def test_invalid_batch_size(self):
         msg = "Batch size must be a positive integer."
         with self.assertRaisesMessage(ValueError, msg):
             Note.objects.bulk_update([], fields=["note"], batch_size=-1)
+            Note.objects.bulk_update([], fields=["note"], batch_size=0)
```
Comment (David-Wobrock) [inline tests/queries/test_bulk_update.py:128]: This line is never executed.
Instead:
```
        with self.assertRaisesMessage(ValueError, msg):
            Note.objects.bulk_update([], fields=["note"], batch_size=0)
```

**Review by David-Wobrock — COMMENTED** [review]
Thanks for submitting a patch
Today, a `batch_size=0` is simply considered the same as `None` in the end, and therefore we will use the `max_batch_size`.
An alternative could be to simply document this behaviour. But I'd be in favour of raising an error in this case 👍

File: tests/queries/test_bulk_update.py (hunk 1 above)
Comment (ebram96) [reply tests/queries/test_bulk_update.py:128]: > But I'd be in favour of raising an error in this case +1

Yep, I prefer an error too. Note: The same file has `bulk_create()` and it raises an error if `batch_size=0`.
Also, the error message won't be consistent with the behavior when a user explicitly passes `batch_size=0` (Intentionally, or, by mistake :D).

> This line is never executed.

Thank you, I'll update it.

**Review by Riahiamirreza — APPROVED** [review]

**Comment by felixxm** [comment]
@ebram96 Thanks :+1: Welcome aboard :boat:

**Pull request merged**